Modules/Core/SpatialObjects/include/itkSurfaceSpatialObject.h

```diff
@@ -71,9 +71,13 @@ class ITK_TEMPLATE_EXPORT SurfaceSpatialObject
   void
   Clear() override;
 
-  /** Compute the normals to the surface from neighboring points */
+/** Compute the normals to the surface from neighboring points */
```

Comment:
This docstring might need to go on line 79.

Reply:
Done.